actions/websocket_actions.jsx

```diff
@@ -825,15 +826,22 @@ export async function handleUserUpdatedEvent(msg) {
     const currentUser = getCurrentUser(state);
     const user = msg.data.user;
 
-    if (isGuest(user)) {
+    const config = getConfig(state);
+    const isTimezoneEnabled = config.ExperimentalTimezone === 'true';
+    const userIsGuest = isGuest(user);
+    if (userIsGuest || isTimezoneEnabled) {
         let members = getMembersInCurrentChannel(state);
         const currentChannelId = getCurrentChannelId(state);
-        if (members && members[user.id]) {
-            dispatch(getChannelStats(currentChannelId));
-        } else {
+        if (!members || !members[user.id]) {
```

Comment:
Could we possibly do something like 
```
const memberExist = members && members[user.id]
if (!memberExist) { ... }
if (memberExist) { ... } 
```
Just thought it may be clearer. What do you think?

Reply:
done